### DISCUSSION TITLE
Notification actions in multitenancy doesn't work

### DISCUSSION THREAD

**Discussion by JBiezais** [laravel/nova-issues#6212] [Ideas & Feature Requests]
- Laravel Version: 10.x
- Nova Version: 4.28.0
- PHP Version: 8.2

### Description:

If Nova is being used for multitenancy web applications, notification actions don't work as expected.

After receiving a notification there are 4 possibilities for what a person can do:

1. Mark All Read
2. Delete All
3. Mark Read
4. Delete

Bulk actions work as expected, but the single action response is "404 Notification not found".

The issue is in NotificationReadController and NotificationDeleteController, parameter $notification is not type hinted therefore $notification value is actually subdomain value.

Request URL is {subdomain}.localhost/nova-api/nova-notifications/{notification} and  {subdomain}.localhost/nova-api/nova-notifications/{notification}/read

**Comment by crynobone** [comment]
Released with Laravel Nova 5.0.0

Feel free to open up a new issue if you're still experiencing this problem on the latest version.
